kpet/data.py

```diff
@@ -262,7 +262,8 @@ def __init__(self, data):
                     match=Class(PositivePattern),
                     dont_match=Class(NegativePattern),
                     url_suffix=String(),
-                    maintainers=NonEmptyList(String())
+                    maintainers=NonEmptyList(String()),
+                    max_runtime_minutes=Int(),
```

Comment:
Do you mind calling it `max_duration_minutes` instead? It's a little easier to read and conjugate and it's only one word.

Reply:
np